nvdaHelper/remote/ia2LiveRegions.cpp

```diff
@@ -213,7 +213,8 @@ bool useIAccessibleForLiveRegionUpdate(IAccessible* pacc, VARIANT varChild) {
 	if(varRole.vt==VT_I4&&varRole.lVal==ROLE_SYSTEM_STATICTEXT) {
 		VariantClear(&varRole);
 		IAccessibleHyperlink* paccHyperlink=NULL;
-		if(pacc->QueryInterface(IID_IAccessibleHyperlink,(void**)&paccHyperlink)==S_OK) {
+		//Note: IAccessibleHyperlink must be QI'ed from IAccessible2
+		if(pacc2->QueryInterface(IID_IAccessibleHyperlink,(void**)&paccHyperlink)==S_OK) {

```

Comment:
Shouldn't the check if not null for pacc2 be moved above this?

Reply:
The function assumes both pacc and pacc2 are non-null so no checks here.